package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "lean",
 	"displayName": "lean",
 	"description": "An extension for VS Code which provides support for the Lean language.",
-	"version": "0.16.20",
+	"version": "0.16.21",
```

Comment:
Please do not increment the version yourself.  I'll do this when I make a release.

Reply:
Oops, sorry!